src/main/java/seedu/duke/atomichabit/atomichabit/AtomicHabitList.java

```diff
@@ -0,0 +1,19 @@
+package seedu.duke.atomichabit.atomichabit;
+
+import java.util.ArrayList;
+
+public class AtomicHabitList {
+    private ArrayList<AtomicHabit> allAtomicHabits;
+
+    public void setAtomicHabits(AtomicHabit atomicHabit) {
```

Comment:
Would addAtomicHabit() be a more appropriate method name? `set` has the connotation that you're replacing/overriding values, which isn't the case here. And this method wraps around the `add()` method of ArrayList, so `add` makes sense. Plus, you're only adding one habit, so `Habits` would be incorrectly plural